guides/common/modules/proc_configuring-dns-dhcp-and-tftp.adoc

```diff
@@ -1,20 +1,11 @@
-[id="configuring-dns-dhcp-and-tftp_{context}"]
+[id="configuring-dns-dhcp-and-tftp-on-productname_{context}"]
```

Comment:
```suggestion
[id="configuring-dns-dhcp-and-tftp_{context}"]
```

"productname" in the anchor feel a bit weird to me. I'd either drop it (violating the "anchor must match title" rule, but OK), or use "{project-context}"/similar.

Reply:
FYI, you have to update the anchor with something different because right now there two same anchors which is causing the build the fail.

Reply:
In the end, I replaced {ProductName} with {ProjectServer} because that seems like the appropriate attribute to use here: https://github.com/theforeman/foreman-documentation/pull/2847/commits/0f1a4de132484a9b873c1980f080b8f55ac0e550